resources/js/layouts/settings/Layout.vue

```diff
@@ -17,6 +18,10 @@ const sidebarNavItems: NavItem[] = [
         title: 'Password',
         href: editPassword(),
     },
+    {
+        title: 'Two-Factor Auth',
+        href: show(),
+    },
```

Comment:
Should "Password" be renamed to "Security" which contains both Password and 2FA stuff?

Reply:
We could combine them, but I feel that doing so would make it harder to remove two-factor authentication section. If someone decides not to use it, it should be easy to delete the related component files/routes and controller.

 Let me know what do you think. @taylorotwell .